README.md

```diff
@@ -17,7 +17,7 @@ Add the following snippet to your project's `pom.xml` file:
 <dependency>
     <groupId>io.harness</groupId>
     <artifactId>ff-java-server-sdk</artifactId>
-    <version>0.0.9.1</version>
+    <version>0.0.10</version>
```

Comment:
Only update this to final version on releasing the sdk. Otherwise it should be a snapshot. When we release a version, tag the release so we can create hotfix on it when required.